python/ray/data/_internal/execution/operators/actor_pool_map_operator.py

```diff
@@ -530,6 +529,8 @@ def __init__(self, max_tasks_in_flight: int = DEFAULT_MAX_TASKS_IN_FLIGHT):
         self._max_tasks_in_flight = max_tasks_in_flight
         # Number of tasks in flight per actor.
         self._num_tasks_in_flight: Dict[ray.actor.ActorHandle, int] = {}
+        # The maximum number of tasks in flight each actor has run at any point in time.
```

Comment:
Should probably avoid non primitive types in stats dict.

Reply:
updated to only report this for the test. This is no longer reported for the user facing API.